modules/features2d/include/opencv2/features2d.hpp

```diff
@@ -247,9 +247,10 @@ class CV_EXPORTS_W BRISK : public Feature2D
     scale 1).
     @param dMin threshold for the long pairings used for orientation determination (in pixels for
     keypoint scale 1).
-    @param indexChange index remapping of the bits. */
+    @param indexChange index remapping of the bits.
+    @param thresh AGAST detection threshold score. */
     CV_WRAP static Ptr<BRISK> create(const std::vector<float> &radiusList, const std::vector<int> &numberList,
-        float dMax=5.85f, float dMin=8.2f, const std::vector<int>& indexChange=std::vector<int>());
+        float dMax=5.85f, float dMin=8.2f, const std::vector<int>& indexChange=std::vector<int>(), int thresh=20);
```

Comment:
Unfortunately you can't change signatures of the public API methods due to binary compatibility restrictions. You can make an overloaded version of `create` without default values for the last two arguments and with reversed order of these ones (`thresh, indexChange`).